pkg/network/server.go

```diff
@@ -347,13 +348,35 @@ func (s *Server) HandshakedPeersCount() int {
 
 // getVersionMsg returns current version message.
 func (s *Server) getVersionMsg() *Message {
+	_, portStr, err := net.SplitHostPort(s.transport.Address())
+	if err != nil {
+		panic(err)
```

Comment:
I added error to return values because we have to handle it somehow. Also if `Transporter` gives as bad address here (e.g. `""`), we'll use server port as TCP capability port.